pynucastro/nucdata/nucleus.py

```diff
@@ -91,6 +92,19 @@ def __init__(self, name, dummy=False):
             self.spec_name = "neutron"
             self.pretty = fr"\mathrm{{{self.el}}}"
             self.caps_name = "n"
+        elif name == "p_nse":
```

Comment:
So, this is the proton used to compute the chemical potential?

Reply:
I don't think it matters, but yes, if we were doing a self-consistent NSE solve on the network, this is the proton we would use, since this is going to be the one associated with the photo-ionization of the heavy nuclei.